source/iot-data-handling/iot-data-handling.rst

```diff
@@ -77,30 +77,7 @@ For each IoT-Device, it is up to the user to select an appropriate payload decod
 
 At the time of writing, there is no retry mechanism in OS2iot over HTTP(s). It uses a "fire-and-forget" / "at most once delivery" pattern.
 
-MQTT data-target
-^^^^^^^^^^^^^^^^
-
-OS2iot supports publishing data to a broker when it's received using MQTT. MQTT is a standard, lightweight messaging protocol based on the publish/subscribe pattern.
-
-When configuring such a data-target, there's a few terms and keywords to be aware of:
-
-- **QoS**: The QoS (Quality of Service) level determines the guarantee of delivery for a specific message. Different network environments may require different QoS levels.
-  Ideally, the level should be set to match the network reliability and application logic. This is the main point of MQTT.
-  
-  There are 3 QoS levels:
-
-  - 0 (at most once)
-  - 1 (at least once)
-  - 2 (exactly once)
-
-  There are a number of well written articles regarding QoS. One such example is
-  `this blog entry <https://www.hivemq.com/blog/mqtt-essentials-part-6-mqtt-quality-of-service-levels/>`_.
-- **Topic**: The MQTT data-target must be provided a topic with which it can label the data. This is used by the MQTT broker to filter messages from
-  MQTT clients. Here, OS2iot is a client.
-- **Connection authentication**: The most common methods of authentication are username and password and/or client certificates. At the time of writing, username and password
-  authentication is supported, but it can be extended to implement other methods. 
-
-You can read more on MQTT `here <https://mqtt.org/>`_
+For more info on the different data-target options, look `here <https://os2iot.readthedocs.io/en/latest/external-interface-design/external-interface-design.html?highlight=data%20target#id2>`_
```

Comment:
Please use "../[end of url] to make the link work no matter if you look at the documentation in Github or readthedocs. For this link it should be <../external-interface-design/external-interface-design.html> (without the highligt details as they are specific to readthedocs)

Reply:
Løses ikke efter aftale.